File: TikTakServer/Models/Video.cs
namespace TikTakServer.Models
{
    public class Video
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string BlobStorageId { get; set; }
        public DateTime UploadDate { get; set; }
        public User User { get; set; }
        public virtual ICollection<Tag> Tags { get; set; }
        public virtual ICollection<Like> Likes { get; set; }
    }
}

File: TikTakClient/src/app/models/video.ts
export interface VideoInfoModel {
    UserId: string;
    ProfileImage: string;
    Email: string;
    Video: Video
  }

  export interface Video {
    BlobStorageId: string;
    UploadDate: string;
    Tags: TagModel[],
    Likes: number
  }

  export interface TagModel {
    Name: string;
  }